yarn.lock

```diff
@@ -2640,6 +2640,14 @@ react-table@^7.6.2:
   resolved "https://registry.yarnpkg.com/react-table/-/react-table-7.6.2.tgz#b60932fa6d457c2bca0da49815cd6a8fe9451f77"
   integrity sha512-urwNZTieb+xg/+BITUIrqdH5jZfJlw7rKVAAq25iXpBPwbQojLCEKJuGycLbVwn8fzU+Ovly3y8HHNaLNrPCvQ==
 
+react-tooltip@^4.2.11:
```

Comment:
Remove this? Looks like you're not touching package.json any more so you can undo your changes here too